LibreOfficeWriter_Char.au3

```diff
@@ -1133,11 +1006,8 @@ EndFunc   ;==>_LOWriter_CharStyleRotateScale
 ; Name ..........: _LOWriter_CharStyleSet
 ; Description ...: Set a Character style for a section of text by Cursor or paragraph Object.
 ; Syntax ........: _LOWriter_CharStyleSet(Byref $oDoc, Byref $oObj, $sCharStyle)
-; Parameters ....: $oDoc           - [in/out] an object. A Document object returned by previous DocOpen, DocConnect, or
-;				   +					DocCreate function.
-;                  $oObj                - [in/out] an object. A Cursor Object returned from any Cursor Object creation
-;				   +						or retrieval functions, Or A Paragraph Object returned from
-;				   +						_LOWriter_ParObjCreateList function.
+; Parameters ....: $oDoc           - [in/out] an object. A Document object returned by previous DocOpen, DocConnect, or DocCreate function.
```

Comment:
Please use the same column location
![image](https://github.com/mlipok/Au3LibreOffice/assets/11089482/fc9bd60f-cbaf-41c8-8ab2-08473d93a61d)